android/run_multiple_test/multiple.conf.js

```diff
@@ -4,20 +4,21 @@ exports.config = {
 
   updateJob: false,
   specs: [
-    './tests/specs/multiple_test.js'
+    './run_multiple_test/specs/multiple_test.js'
   ],
   exclude: [],
 
   capabilities: [{
     name: 'multiple_appium_test',
```

Comment:
follow naming conventions and all capabilities required from https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json